app/views/devise/registrations/edit.html.erb

```diff
@@ -38,6 +38,6 @@
 
 <h3>Cancel my account</h3>
 
-<p>Unhappy? <%= button_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?" }, method: :delete %></p>
+<p>Unhappy? <%= button_to "Cancel my account", registration_path(resource_name), data: { confirm: "Are you sure?", turbo_confirm: "Are you sure?" }, method: :delete %></p>
```

Comment:
In Changelog or Wiki, it might be worth mentioning this change like OmniAuth "Sign in with" links.

Reply:
This seems do not solve #5448. See here: https://github.com/heartcombo/devise/issues/5448#issuecomment-1457338283